src/src-assets/player-assets/player.coffee

```diff
@@ -439,8 +435,6 @@ angular.module('Adventure', ['ngAria', 'ngSanitize'])
 			# Loop through each match to see if it matches the recorded response
 			for j in [0...$scope.q_data.answers[i].options.matches.length]
 
-
-				# TODO make matching algo more robust
 				match = $scope.q_data.answers[i].options.matches[j]
 
 				# Remove whitespace
```

Comment:
This is another error irrelevant to the PR's changes, but would be good to include. The following code is removing special characters in addition to whitespace, leaving only (a-z, A-Z), digits (0-9), and underscores (_), so the character sensitivity option doesn't work properly. We should probably use the replace() function instead with the `\s` metacharacter.

Reply:
Your suggestion appears to do the trick, fixed.